module/zfs/dmu_zfetch.c

```diff
@@ -48,9 +48,13 @@ static int zfs_prefetch_disable = B_FALSE;
 /* max # of streams per zfetch */
 static unsigned int	zfetch_max_streams = 8;
 /* min time before stream reclaim */
-static unsigned int	zfetch_min_sec_reap = 2;
-/* max bytes to prefetch per stream (default 8MB) */
-unsigned int	zfetch_max_distance = 8 * 1024 * 1024;
+static unsigned int	zfetch_min_sec_reap = 1;
+/* max time before stream delete */
+static unsigned int	zfetch_max_sec_reap = 2;
+/* min bytes to prefetch per stream (default 4MB) */
+unsigned int	zfetch_min_distance = 4 * 1024 * 1024;
```

Comment:
this can be static AFAICT?

Reply:
Indeed. I decided to not create FreeBSD compat shim for it, since it was never present there.  So yes it can be static.  Fixed.